futures/pubspec.yaml

```diff
@@ -1,2 +1,6 @@
 name: futures
 description: A sample application
+
+homepage: https://www.dartlang.org/docs/tutorials/futures/
+gist: https://gist.github.com/Sfshaza/9e43fdcdc8848cd1490b
+dartpad: https://dartpad.dartlang.org/9e43fdcdc8848cd1490b
```

Comment:
This is a dart:io sample, so kill the dartpad link. I guess it's OK to keep the gist link?

Maybe we should file an issue on generator-gist asking not to create the dartpad link if the file relies on dart:io or packages?